mlog/default.go

```diff
@@ -4,9 +4,13 @@
 package mlog
 
 import (
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
+
+	"github.com/wiggin77/logr"
```

Comment:
Makes sense.  I'll look into transfering it.

Reply:
I am admin on github. If you transfer it to me, I can transfer it into the org for you.

Reply:
Done

Reply:
And done:
https://github.com/mattermost/logr

Reply:
I updated mattermost/logr so that go.mod has the new moniker, and fixed all the imports.  Also updated this PR to use mattermost/logr, updated all imports, and re-vendored.  wiggin77/logr is gone from vendor.